xcode/Subconscious/Shared/Models/Slashlink.swift

```diff
@@ -224,6 +224,15 @@ extension Slashlink {
             return self
         }
     }
+    
+    func relativizeIfNeeded(petname base: Petname?) -> Slashlink {
+        switch self.peer {
+        case .petname(let name) where name == base:
+            return Slashlink(slug: self.slug)
+        default:
+            return self
+        }
+    }
```

Comment:
Similar to the DID version, drop the peer from the address if it matches the base.